src/pages/casbin-editor.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+function Editor() {
+    return (
+        <iframe
+            src="https://casbin.github.io/casbin-editor/"
+            style={{ width: '100%', height: '100vh', display: 'block' }}
+            title="Casbin-editor"
+            allowFullScreen={true} />
+    );
+}
+
+export default Editor;
```

Comment:
Add a page to simulate [Casbin Online Editor](https://casbin.org/casbin-editor/)
Preview: https://deploy-preview-12--casbin.netlify.app/casbin-editor/